### PR TITLE
Fix a few small issues

### PR DESCRIPTION
**Pull request by Heaven31415**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -166,7 +166,7 @@ To directly bind your data to your custom types, you need to specify the
 variable name to be exactly the same as the key in the JSON document returned from the API.
 In case your variable name and key in JSON doc do not match, you can use `@JsonProperty`
 annotation to specify the exact key of the JSON document. (This example matches each
-variable name to a JSON key, so do not need that annotation here.)
+variable name to a JSON key, so we do not need that annotation here.)
```
Comment (dsyer) [inline README.adoc]: Guides all should be using direct, imperative speech ("you" no "we"). We (sic) might as well try and stick to that.

File: README.adoc [hunk 2]
```diff
@@ -188,7 +188,7 @@ class the Initializr creates (at
 `src/main/java/com/example/consumingrest/ConsumingRestApplication.java`):
 
 ====
-[src,Java]
+[source,java,tabsize=2]
```
Comment (dsyer) [inline README.adoc]: The lower case is correct. We prefer not to replace tabs with spaces for accessibility reasons.

File: README.adoc (hunk 1 above)
Comment (Heaven31415) [reply README.adoc]: So 'you' instead of 'we' and you are going to be happy, right?

File: README.adoc (hunk 2 above)
Comment (Heaven31415) [reply README.adoc]: Basically I wanted to fix syntax highlighting which looks like this right now: 

image

I found that `[source,java,tabsize=2]`  from `README.adoc` is working, but if you want I can delete the **tabsize** option. 

Are you going to be happy with `[source,java]`?

File: README.adoc (hunk 1 above)
Comment (dsyer) [reply README.adoc]: Yes, that's what I meant.

File: README.adoc (hunk 2 above)
Comment (dsyer) [reply README.adoc]: > Are you going to be happy with [source,java]?

Yes, that's what I meant. The highlighting is broken by the "J" not the tabsize.

**Pull request merged**